jd4/config.py

```diff
@@ -5,17 +5,22 @@
 from os import path
 from ruamel import yaml
 
+from jd4.log import logger
+
 _CONFIG_DIR = user_config_dir('jd4')
 _CONFIG_FILE = path.join(_CONFIG_DIR, 'config.yaml')
 _COOKIES_FILE = path.join(_CONFIG_DIR, 'cookies')
 
 def _load_config():
+    if not path.exists(_CONFIG_FILE):
```

Comment:
This should be implemented as try...except FileNotFoundError. otherwise there is a time-of-check-to-time-of-use problem.

Also this was intended - the error message generated by the uncaught exception seems fine to me.